plugins/scaffolder-backend/src/scaffolder/stages/publish/github.ts

```diff
@@ -54,6 +55,27 @@ export class GithubPublisher implements PublisherBase {
 
     const { data } = await repoCreationPromise;
 
+    if (access.match(new RegExp(`${owner}/.+`))) {
+      const [, team] = access.split('/');
+      await this.client.teams.addOrUpdateRepoPermissionsInOrg({
```

Comment:
What happens if this repo is not in an organisation?

Reply:
Only teams come through this flow, org/team.

I can add this check if you think it's worth it?

`user.data.type === 'Organization'`

but it will only happen by accident and don't think we have the validation implemented yet above json-schema so can't really feed back to the client right now.

Reply:
Hmmm, 
Sorry I thought that the difference in this ternary might make this operation fail no?
https://github.com/spotify/backstage/blob/7806decbc4e19f4338600695a177efdea24f1e87/plugins/scaffolder-backend/src/scaffolder/stages/publish/github.ts#L51-L58

Reply:
Individual users can't have teams, so the access.match should only be validating against an individual user.

i.e. this case shouldn't happen unless a user is trying it to see what happens, or accidentally is trying to create it in their account.

Reply:
ah ok, now I understand. Sweet.